doc/Changelog.md

```diff
@@ -1,8 +1,12 @@
 Changelog {#Changelog}
 =========
 
-# Release 2.0.0 (24-May-2017)
+# Release 2.0.0 (30-May-2017)
 
+* [157](https://github.com/BlueBrain/Brion/pull/157):
+  Add Synapse.gid() python wrapping
+* [157](https://github.com/BlueBrain/Brion/pull/157):
```

Comment:
repeated 157? In this chagelog we use bullet points for PRs with multiple changes. I've never seen the other style in any other project.